src/token/components/TokenRow.tsx

```diff
@@ -23,9 +23,7 @@ export const TokenRow = memo(function TokenRow({
         <span className="flex flex-col items-start">
           <TextHeadline>{token.name}</TextHeadline>
           {!hideSymbol && (
-            <span className="font-normal text-[#5B616E] leading-normal">
-              {token.symbol}
-            </span>
+            <TextBody variant="foreground-muted">{token.symbol}</TextBody>
```

Comment:
I do not consider this an improvement in DevX. I can read and immediately know what the former implementation looks like, but the new version requires me to look into `TextBody` and our variant mapping to figure out what this looks like.

Reply:
The point is that you do not have to read what `TextBody` is.

Reply:
That is to say, we can rename `variant` to `color` instead to make it clear what it affects.

Reply:
I still don't think that autocomplete of a color prop is worth adding the abstraction versus just doing the raw className. It feels like more juice than the squeeze and we have to maintain this moving forward. I'll lean on your judgement, but my opinion is to be ruthless with adding new abstractions.

Reply:
Oh you better know I hate meaningless abstractions haha

Reply:
I would say that once you have component name + color autocompletion _and_ you have minimal abstraction between understanding designs and converting them to code, you don't want to go back. I promise you.

Reply:
My summary here is that abstraction are not need it as much in this case when the team is small, and also the codebase is small.

But as soon as you get 3+ eng and 2+ designers, and 20+ pages with 100+ components. Any meaninful re-design or design standardization, it's faster and easier when abstraction like this one are in place.

Coinbase had not this kind of abstraction in 2019 and back than was a pain align on desing or any meaniful refactoring. When we adopted a more mature Design System in 2020, that allowed us to re-shape 100s of pages in just weeks and also keep high level of consistency between all those pages.

So again, a team with just 3~4 people, with a small codebase, they probably don't need this. Anything larger, they will appreciate this, as it will increase quality and speed overall.